torch/distributed/fsdp/_runtime_utils.py

```diff
@@ -548,6 +593,15 @@ def _post_backward_hook(
                     padded_unsharded_grad,
                     new_sharded_grad,
                 )
+                if handle._config.sharding_strategy in {
```

Comment:
nit: No point incurring the cost of constructing a `set` here if there are only 2 elements. `tuple` is fine.

Reply:
[Easy] I recommend changing before landing